docs/cnfc.md

```diff
@@ -0,0 +1,35 @@
+# CNFC
+
+Keycloak is currently a CNCF incubating project. This document provides useful information and links around CNFC.
+
+## Useful links
+
+* [CLO Monitor](https://clomonitor.io/projects/cncf/keycloak)
+* [CNCF Artwork](https://github.com/cncf/artwork/tree/master/projects/keycloak)
+* [CNCF Landscape](https://landscape.cncf.io/?selected=keycloak)
+* [CNCF Online Programs Guidelines](https://github.com/cncf/foundation/blob/main/online-programs-guidelines.md)
+* [CNFC Service Desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portals)
+* [DevStats](https://keycloak.devstats.cncf.io/)
+* [Slack](https://cloud-native.slack.com/) (current channels #keycloak, #keycloak-dev, and #keycloak-maintainers)
+* [Services for CNCF Projects](https://www.cncf.io/services-for-projects/)
+* [Schedule time with the CNCF Projects Team](http://project-meetings.cncf.io/)
+
+## Maintainers
```

Comment:
Should we also add a link to https://github.com/keycloak/keycloak/blob/main/GOVERNANCE.md#maintainers?

Reply:
I'd say not, as this is links related to CNCF, not general links for the project